src/Entities/ArrowEntity.cpp

```diff
@@ -3,28 +3,33 @@
 #include "Player.h"
 #include "ArrowEntity.h"
 #include "../Chunk.h"
+#include "../ClientHandle.h"
+
+
+
+
+
+/** Converts an angle in radians into a byte representation used by the network protocol */
+#define ANGLE_TO_PROTO(X) static_cast<Byte>(X * 255 / 360)
```

Comment:
Yaw and Pitch are stored in degrees in the entity (wrong comment)

Reply:
I've created an issue to move this directly into `cProtocol` descendants: #4679.